packages/tradershub/src/features/cfd/screens/Jurisdiction/JurisdictionCard/JurisdictionCard.tsx

```diff
@@ -55,14 +54,14 @@ const verificationDocumentsMapper: TVerificationDocumentsMapper = {
 const JurisdictionCard = ({
     isAdded = false,
     isSelected = false,
-    jurisdiction = 'svg',
+    jurisdiction = Jurisdiction.SVG,
     onSelect,
 }: TJurisdictionCardProps) => {
     const [isFlipped, setIsFlipped] = useState(false);
     const { toggleDynamicLeverage } = useDynamicLeverageModalState();
     const { getCFDState } = Provider.useCFDContext();
 
-    const descriptionClickHandler = (tag?: TJurisdictionCardProps['tag']) => (event: MouseEvent) => {
```

Comment:
Why removed this type?

Reply:
The tag in the function params is different, it's not coming from props, and we are not using 'tag' prop anywhere.